### PR TITLE
Pretty flash alert/notice + disambiguity

### LINKED ISSUE #120: make flash notifications pretty

### PR DESCRIPTION
**Pull request by StoveSolder**
Creates dismissable and prettier alerts.

Test plan:

1. Attempt action which prompts a notice/alert (Upload users via csv, restore a user, etc)
2. See pretty notification appear, click to dismiss

Close #120

### CODE REVIEW

File: app/assets/javascripts/alert.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+!function ($) {
+
+    "use strict"; // jshint ;_;
+  
+  
+   /* ALERT CLASS DEFINITION
+    * ====================== */
+  
+    var dismiss = '[data-dismiss="alert"]'
```
Comment (BlinkVonDoom) [inline app/assets/javascripts/alert.js]: change `var`s to `let`s

File: app/assets/javascripts/alert.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      , Alert = function (el) {
```
Comment (BlinkVonDoom) [inline app/assets/javascripts/alert.js]: not necessary but look into arrow functions

File: app/assets/javascripts/alert.js (hunk 1 above)
Comment (StoveSolder) [reply app/assets/javascripts/alert.js]: `var`s changes to `let`s as  `let` is a better signal that the variable may be reassigned and scopes better

File: app/assets/javascripts/alert.js (hunk 2 above)
Comment (StoveSolder) [reply app/assets/javascripts/alert.js]: based on my limited understanding and this flow chart (linked in the article), https://github.com/getify/You-Dont-Know-JS/blob/master/es6%20%26%20beyond/fig1.png I'm going to not include the arrow functions in this code. However, in future content, i will look for more opportunities to use Arrow functions where appropriate, as they can make many chained functions much easier to read.

File: app/assets/javascripts/alert.js (hunk 1 above)
Comment (StoveSolder) [reply app/assets/javascripts/alert.js]: fixed by 
b5c01bc

**Pull request merged**